.github/workflows/fw_icu4c_ci.yml

```
@@ -161,25 +161,25 @@ jobs:
           cd "${WORKSPACE}/"
           version="${{needs.set-version-number.outputs.version}}${{ env.package_suffix }}"
           ${CI_SCRIPT_DIR}/bash/make-source \
-            --dists "xenial focal bionic" \
+            --dists "noble xenial focal bionic" \
```

We should remove the EOL versions (xenial, bionic) and focal which will be EOL in April.

```suggestion
            --dists "noble jammy" \
```